service/notifier.go

```diff
@@ -318,3 +335,37 @@ func (p *notifier) sendMessageToWebhook(notification *Notification, webhook *con
 	p.logger.Debug(fmt.Sprintf("response Body: %v", string(body)))
 	return nil
 }
+
+func Retry(operation backoff.Operation, policy backoff.RetryPolicy, isRetryable backoff.IsRetryable) error {
```

Comment:
Can you expand on this why we need to implement our own retry?
Also it doesn't seem need to be public(Capitalized)